1Password/op-scim-helm
# Update chart name and remove legacy Chart packages

In this PR we update the Chart name from `op-scim` to `op-scim-helm`. We also make the necessary changes to `values.yaml` so that the actual deployed artifacts are still called `op-scim-helm-*`, i.e. same as before.

We remove the legacy Chart packages located in the root of the project. These no longer work with the changes made to the `index.yaml` at the root of the project. The `index.yaml` is kept around for now to provide additional guidance to folks looking to upgrade to the new repo URL.

### Steps to test

1. Install the previous version of the Chart (i.e. v2.4.2) from the repo:
    ```
    helm install op-scim-bridge 1password/op-scim --create-namespace --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=standard
    ```
    - Remember to select the correct storage class for your destination platform, i.e. `standard` for GCP and `do-block-storage` for DO
2. Ensure that the applications are deployed successfully and that there are no errors in the logs:
    - `kubectl get pods --namespace=op-scim-bridge -o wide`
    - `kubectl logs <op-scim-bridge-pod> --namespace=op-scim-bridge`
3. Package the latest chart changes: `helm package charts/op-scim-bridge`
4. Update using the latest chart changes:
    ```
    helm upgrade op-scim-bridge op-scim-bridge-2.5.0.tgz --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=standard
    ```
5. Once again ensure that the applications are deployed successfully and that there are no errors in the logs:
    - `kubectl get pods --namespace=op-scim-bridge -o wide`
    - `kubectl logs <op-scim-bridge-pod> --namespace=op-scim-bridge`
6. [optional] Try the latest chart changes as a fresh install:
    1. Uninstall the previous configuration `helm uninstall op-scim-bridge --namespace=op-scim-bridge`
    2. Ensure no application pods are deployed or running: `kubectl get pods --namespace=op-scim-bridge -o wide`
    3. Install (not upgrade) the latest chart changes:
        ```
        helm install op-scim-bridge op-scim-bridge-2.5.0.tgz --create-namespace --namespace op-scim-bridge --set scim.credentialsVolume.storageClass=standard
        ```
    4. Once again confirm that there are no errors in the deployments or application logs

Resolves #79.
Resolves #69.
